test/src/main/scala/org/scalatra/test/Client.scala

```diff
@@ -94,10 +94,14 @@ trait Client extends ImplicitConversions {
     patch(uri, params)(f)
   def patch[A](uri: String, params: Iterable[(String, String)])(f: => A): A =
     patch(uri, params, Map[String, String]())(f)
-  def patch[A](uri: String, params: Iterable[(String, String)], headers: Iterable[(String, String)])(f: => A): A =
+  def patch[A](uri: String, params: Iterable[(String, String)], headers: Map[String, String])(f: => A): A =
```

Comment:
Yes, after adding the new method it wouldn't compiler without this change. I think I was getting ambiguity error.